File: socialuni-server/socialuni-community-sdk/src/main/java/com/socialuni/social/community/sdk/model/SocialCircleRO.java
package com.socialuni.social.community.sdk.model;

import lombok.Data;

/**
 * @author qinkaiyuan
 * @date 2019-11-07 15:20
 */

@Data
public class SocialCircleRO {
    public Integer id;
    public String name;
    public String avatar;
    public String description;
    public Boolean selected;
    private Integer talkCount;
    private Integer count;
}

File: socialuni-js/packages/socialuni-base/src/model/community/circle/SocialCircleRO.ts
export default class SocialCircleRO {
  public id: number = null
  public name: string = null
  //群聊id
  public groupId: string = null
  public selected = false
  public count: number = null
  public talkCount: number = null
  public avatar: string = null
  public description: string = null


  constructor (name?: string) {
    this.name = name || null
  }
}
